indigo_app/templates/indigo_app/place/_work.html

```diff
@@ -25,7 +25,7 @@
           <div class="d-flex">
             <div class="col-9 ps-0">
               <div>
-                <a href="{% url 'work' frbr_uri=work.frbr_uri %}"><span>{{ work.title }}</span></a>
+                <a href="{% url 'work' frbr_uri=work.frbr_uri %}">{{ work.title }}</a>
```

Comment:
```suggestion
                <a href="{% url 'work' frbr_uri=work.frbr_uri %}"><span>{{ work.title }}</span></a>
```

I know this sounds crazy, but that span is what's making the link actually clickable versus just expanding and collapsing the card (see https://www.loom.com/share/512c3599cf534efc8f4bc172bf864188)

Reply:
woah, it seems to be related to how data-bs-toggle works to cancel the event if it comes from `<a>`